sycl/plugins/unified_runtime/pi2ur.hpp

```diff
@@ -2969,6 +2896,28 @@ inline pi_result piextUSMDeviceAlloc(void **ResultPtr, pi_context Context,
   return PI_SUCCESS;
 }
 
+inline pi_result piextUSMPitchedAlloc(void **ResultPtr, size_t *ResultPitch,
+                                      pi_context Context, pi_device Device,
+                                      pi_usm_mem_properties *Properties,
+                                      size_t WidthInBytes, size_t Height,
+                                      unsigned int ElementSizeBytes) {
+  PI_ASSERT(Context, PI_ERROR_INVALID_CONTEXT);
+  PI_ASSERT(Device, PI_ERROR_INVALID_DEVICE);
+
+  ur_context_handle_t UrContext =
+      reinterpret_cast<ur_context_handle_t>(Context);

```

Comment:
I think `auto UrName reinterpret_cast<ur_type>...` reads better, especially for mouthful types like `ur_context_handle_t`, but unfortunately this file already has a mixture of `auto` and explicit types.